src/extensions/storage_macro.rs

```diff
@@ -2,6 +2,8 @@ mod append;
 mod chunk;
 mod fs;
 mod redis;
+mod dbc;
+
```

Comment:
Could you unify the mod name(`dbc`) and macro type(`rdbc`)? Using the same name will be helpful. Thanks!

Reply:
Done, made everything "dbc"